packages/shared/src/components/layout/MainLayoutHeader.tsx

```diff
@@ -1,7 +1,6 @@
 import classNames from 'classnames';
 import React, { ReactElement, ReactNode, useCallback } from 'react';
 import dynamic from 'next/dynamic';
-import { useAuthContext } from '../../contexts/AuthContext';
```

Comment:
Feels like we can break this component to multiple smaller ones, though I think that is for another PR.